pkg/cmd/server/origin/leaderelection.go

```diff
@@ -81,10 +87,6 @@ func NewLeaderElection(options configapi.MasterConfig, leader componentconfig.Le
 
 	// use the endpoints leader election path.
 	plug := plug.New(false)
-	events := record.NewBroadcaster()
-	events.StartLogging(glog.Infof)
-	events.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(kc.Core().RESTClient()).Events("")})
-	lock.LockConfig.EventRecorder = events.NewRecorder(kapi.Scheme, v1corev1.EventSource{Component: name})
 	elector, err := leaderelection.NewLeaderElector(leaderelection.LeaderElectionConfig{
```

Comment:
Yeah, that complexity was in a previous version of the PR.  I think I'm going to pull it out, and require the ansible upgrade process to stop all controllers, change the config, then start the new controllers.